rest_framework/validators.py

```diff
@@ -211,7 +214,9 @@ def __call__(self, attrs):
         queryset = self.exclude_current_instance(attrs, queryset)
         if queryset.exists():
             message = self.message.format(date_field=self.date_field)
-            raise ValidationError({self.field: message})
+            raise ValidationError({
+                self.field: message,
+            })
```

Comment:
To add to my previous comment, this code show why having the code here is useless:
https://github.com/tomchristie/django-rest-framework/blob/master/rest_framework/fields.py#L502